src/Featurizers/Components/DocumentStatisticsEstimator.h

```diff
@@ -137,8 +240,9 @@ class DocumentStatisticsTrainingOnlyPolicy {
     // |  Public Methods
     // |
     // ----------------------------------------------------------------------
-    DocumentStatisticsTrainingOnlyPolicy(void);
-    DocumentStatisticsTrainingOnlyPolicy(StringDecorator decorator);
+    DocumentStatisticsTrainingOnlyPolicy(StringDecorator decorator, nonstd::optional<IndexMap> vocabulary, nonstd::optional<std::uint32_t> maxFeatures, std::float_t minDf, std::float_t maxDf);
```

Comment:
Can you change the name of vocabulary to indicate that this is the vocabulary produced as part of a previous training session? Maybe "existingVocabulary"?

Reply:
sure